### ISSUE TITLE
PHPUnit 6 compatibility

### ISSUE THREAD

**Issue by sergeyz** [jwage/phpchunkit#45]
We have to changes parent classes

```
Fatal error: Class 'PHPUnit_Framework_TestCase' not found in /home/travis/build/jwage/phpchunkit/tests/BaseTest.php
```

https://travis-ci.org/jwage/phpchunkit/jobs/199652306

**Comment by jwage** [comment]
How could we make it work for both versions?

**Comment by sergeyz** [comment]
@jwage We can try to use PHPUnit compatibility layer or do class_alias if it's not possible

**Comment by jwage** [comment]
Do you want to try and make a pull request?

**Comment by mcneely** [comment]
`
if (!class_exists('\PHPUnit\Framework\TestCase') &&
    class_exists('\PHPUnit_Framework_TestCase')) {
    class_alias('\PHPUnit_Framework_TestCase', 'PHPUnit\Framework\TestCase');
}
`

**Comment by jwage** [comment]
Fixed in #51
